js/widgets/status.js

```diff
@@ -12,20 +12,22 @@
 // This widget makes displays the status of selected parameters and
 // notes as they are being played.
 
-function StatusMatrix() {
-    const BUTTONDIVWIDTH = 128;
-    const BUTTONSIZE = 53;
-    const ICONSIZE = 32;
-    const OUTERWINDOWWIDTH = 620;
-    const INNERWINDOWWIDTH = OUTERWINDOWWIDTH - BUTTONSIZE * 1.5;
-    const FONTSCALEFACTOR = 75;
-    let x, y; //Drop coordinates of statusDiv
-
-    this.init = (logo) => {
+class StatusMatrix {
+    constructor() {
+        this.BUTTONDIVWIDTH = 128;
+        this.BUTTONSIZE = 53;
+        this.ICONSIZE = 32;
+        this.OUTERWINDOWWIDTH = 620;
+        this.INNERWINDOWWIDTH = OUTERWINDOWWIDTH - BUTTONSIZE * 1.5;
+        this.FONTSCALEFACTOR = 75;
+        let x, y; //Drop coordinates of statusDiv
+    }
+
+    init = () => {
         // Initializes the status matrix. First removes the
         // previous matrix and them make another one in DOM (document
         // object model)
-        this._logo = logo;
+
         this.isOpen = true;
 
         let w = window.innerWidth;
```

Comment:
Please use `const` over `let` for variables which are not being re-assigned. Do this across the file. Pull the latest changes then try linting the file with ES-Lint to find such issues. We have enabled `prefer-const` rule in ES-Lint recently.

Reply:
okay.

Reply:
There are many variables that are not initialized in the file. So, for them I need to used let as using const gives error in ESLint?
Do you suggest anything else?

Reply:
Using const would give error if it is being reassigned. If the variable is not being reassigned then use of const should be preferred over let. All variables need to be defined using either let or const before they are used.